components/SnippetCardList.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import SnippetCard from "./SnippetCard";
+
+export default function SnippetCardList({ data }) {
+  return (
+    <>
+      <ListHeading>List of Snippets</ListHeading>
+      {data.map((snippet) => (
+        <p key={snippet._id}>
+          <SnippetCard
+            snippetData={snippet}
+            name={snippet.name}
+            description={snippet.description}
+          />
+        </p>
+      ))}
```

Comment:
Working with a list and list items would make this part of your code more semantic:

```suggestion
       <ul>
      {data.map((snippet) => (
        <li key={snippet._id}>
          <SnippetCard
            snippetData={snippet}
            name={snippet.name}
            description={snippet.description}
          />
        </li>
      ))}
     </ul>
```